src/stories/PopoverMenu.stories.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import {SafeAreaProvider} from 'react-native-safe-area-context';
+import PopoverMenu from '../components/PopoverMenu';
+import * as Expensicons from '../components/Icon/Expensicons';
+import MenuItem from '../components/MenuItem';
+import colors from '../styles/colors';
+
+/**
+ * We use the Component Story Format for writing stories. Follow the docs here:
+ *
+ * https://storybook.js.org/docs/react/writing-stories/introduction#component-story-format
+ */
+const story = {
+    title: 'Components/PopoverMenu',
+    component: PopoverMenu,
+};
+
+// eslint-disable-next-line react/jsx-props-no-spreading
+const Template = (args) => {
+    const [isVisible, setIsVisible] = React.useState(false);
+    const toggleVisibility = () => setIsVisible(!isVisible);
+    return (
+        <>
+            <MenuItem
+                // eslint-disable-next-line react/jsx-props-no-spreading
+                {...args.menuItem}
+                onPress={toggleVisibility}
+                wrapperStyle={isVisible ? [{backgroundColor: colors.gray2}] : []}
+            />
+            <SafeAreaProvider>
+                <PopoverMenu
+                    // eslint-disable-next-line react/jsx-props-no-spreading
+                    {...args.popoverMenu}
+                    isVisible={isVisible}
+                    onClose={toggleVisibility}
```

Comment:
@thesahindia please make sure all the required props are passed for PopOverMenu.
Refer https://github.com/Expensify/App/blob/main/src/components/PopoverMenu/popoverMenuPropTypes.js